doc/ref_model/chapters/chapter03.rst

```diff
@@ -365,15 +365,15 @@ This section proposes a Redfish based hardware infrastructure manager for the An
 Chapter 9 (:ref:`chapters/chapter09:Configuration and Lifecycle Management`) of this reference model specifies Redfish standard for managing Infrastructure Hardware. The GSMA's "Cloud Infrastructure Reference Model (NG.126)" also specifies Redfish as the standard interface that should be 
 made available by the infrastructure and Cloud Infrastructure Management components, in the "Infrastructure Hardware" layer.
  
-Redfish is an internationally recognized standard `ISO/IEC 30115:2018  <https://www.iso.org/standard/83853.html>`__. The Redfish interface specifies a HTTP RESTful interface that a client can use to manage conformant platforms. The Redfish standard consists of a Redfish interface specification `Redfish Specification <https://www.dmtf.org/sites/default/files/standards/documents/DSP0266_1.8.0.pdf>`__ and model specification `Redfish Data Model Specification <https://www.dmtf.org/sites/default/files/standards/documents/DSP0268_2022.2.pdf>`__.  The interface specification defines the RESTful behavior of the resources. The data model specification defines the structure of the HTTP resources. The model is expressed as schema using OpenAPI and json-schema formats. The schema allows implementing redfish clients using the OpenAPI and json-schema toolchains.
+Redfish is an internationally recognized standard ISO/IEC 30115:2018 :cite:p:`iso-standard-83853`. The Redfish interface specifies a HTTP RESTful interface that a client can use to manage conformant platforms. The Redfish standard consists of a Redfish interface specification `Redfish Specification <://www.dmtf.org/sites/default/files/standards/documents/DSP0266_1.8.0.pdf>`__ and model specification Redfish Data Model Specification :cite:p:`redfish-data-model-specification`.  The interface specification defines the RESTful behavior of the resources. The data model specification defines the structure of the HTTP resources. The model is expressed as schema using OpenAPI and json-schema formats. The schema allows implementing redfish clients using the OpenAPI and json-schema toolchains.
```

Comment:
Move the "Redfish Specification" also to the bib file and fix the typo in the link.

Reply:
Updated